src/applications/simple-forms/20-10206/config/constants.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 
+export const TITLE = 'Request personal records';
+export const SUBTITLE =
+  'Freedom of Information Act (FOIA) or Privacy Act (PA) Request (VA Form 20-10206)';
```

Comment:
was it intentional to change the wording from "and" to "or"?

Reply:
yep.  i double-checked the mockups.